mimc7/mimc7.go

```diff
@@ -23,22 +23,22 @@ type constantsData struct {
 }
 
 func generateConstantsData() constantsData {
-	var constants constantsData
+	var consts constantsData
 
-	constants.seedHash = new(big.Int).SetBytes(keccak256.Hash([]byte(SEED)))
+	consts.seedHash = new(big.Int).SetBytes(keccak256.Hash([]byte(SEED)))
 	c := new(big.Int).SetBytes(keccak256.Hash([]byte(SEED + "_iv")))
-	constants.iv = new(big.Int).Mod(c, _constants.Q)
+	consts.iv = new(big.Int).Mod(c, _constants.Q)
```

Comment:
Could you rename `_constants` to `constants`?

Reply:
there's already global var with this name